models/Staticroute.php

```diff
@@ -696,6 +701,38 @@ public function match($path, $params = [])
         return [];
     }
 
+
+    /**
+     * @return array
+     */
+    public function getMethods()
+    {
+        if ($this->methods && !is_array($this->methods)) {
+            $this->methods = explode(',', $this->methods);
+        }
+
+        return $this->methods;
+    }
+
+    /**
+     * @param array $methods
+     *
+     * @return $this
+     */
+    public function setMethods($methods)
+    {
+        if (!is_array($methods)) {
+            $methods = strlen($methods) ? explode(',', $methods) : [];
+            $methods = array_map(static function ($entry) {
```

Comment:
Why not just `$methods = array_map('trim', $methods );` ?

Reply:
because it really doesn't matter.